mapstore2_adapter/plugins/geonode.py

```diff
@@ -259,17 +138,116 @@ def convert(self, viewer, request):
                 ms2_map_data = ms2_resource.data.blob
                 if isinstance(ms2_map_data, string_types):
                     ms2_map_data = json.loads(ms2_map_data)
-                if 'map' in ms2_map_data:
-                    for _k, _v in ms2_map_data['map'].items():
-                        if _k not in data['map']:
-                            data['map'][_k] = ms2_map_data['map'][_k]
-                    del ms2_map_data['map']
-                data.update(ms2_map_data)
+                data = ms2_map_data
```

Comment:
@marthamareal the only thing that we should override from the original bundle are the backgrounds. If backgrounds are changed inside GeoNode settings they wouldn't be picked by the map.
It should be straightforward, since background layers are assigned to the "background" group. You should cycle over the layers, remove the layers inside the backgrounds group and add the backgrounds layer back according to GeoNode settings. 
You can take inspiration from[ this block of code](https://github.com/GeoNode/geonode-mapstore-client/blob/66bf4a919bf0b048e38bf9787e8d1cdc90a423cf/mapstore2_adapter/plugins/geonode.py?plain=1#L282-L290).

Reply:
@giohappy  I have updated the PR with the suggested changes